src/IPub/MobileDetect/MobileDetect.php

```diff
@@ -63,16 +63,6 @@ public function __construct(
 	}
 
 	/**
-	 * Check if the device is mobile phone
-	 *
-	 * @return bool
-	 */
-	public function isPhone() : bool
```

Comment:
Tohle bych určitě nechtěl rušit, ti co tuhle knihovnu používají na tomhle určitě stavějí. Spíše by se měla upravit aby vracela validní stav

Reply:
Ahoj, BC je 100% zaručena. Funkce je zrušena, protože ji již implementuje to rozšíření nad mobile detectem.

Reply:
aaa jsný, já toho agenta zatím moc nestudoval, jen lehce proletěl. 👍